README.md

```diff
@@ -101,6 +110,25 @@ Set these environment variables when debugging:
 RUST_LOG=lorri=debug RUST_BACKTRACE=1 lorri watch
 ```
 
+### `lorri` reevaluates more than expected
+
+`lorri` sometimes recursively watches a directory that the user did
+not expect. This can happen for a number of reasons:
+
+1. when using a directory instead of a channel for `nixpkgs`, `lorri`
+   watches that directory recursively, including the potential `.git`
+   directory and its subdirectories
```

Comment:
`when using a local checkout instead`

I think the second part of the sentence is redundant, but you could say `, and will trigger on any file change`.